docs/community/local-groups.mdx

```diff
@@ -88,6 +88,11 @@ us on [Discord](https://discord.com/invite/ktMAKGBnBs) to add your group.
 - [Mesh Finland Discord](https://discord.com/invite/GHnaVAjqed)
 - [Mesh Finland Website](https://mesh-finland.github.io)
 
+## France
+
+- [Meshtastic France - Facebook](https://www.facebook.com/groups/meshtastic.france)
```

Comment:
The name of this one will need to change to something like Mesh France, Meshtastic France Club, or something else that makes it clear this isn't an official Meshtastic entente.

We have some docs and guidelines around this here: https://meshtastic.org/docs/legal/licensing-and-trademark/#noncommercial-and-community-web-sites

Reply:
I made the change as requested.

Reply:
@Raxgahrax thanks for updating it here on the link. I am still seeing the name as the same on the actual page though.

<img width="1253" alt="Screenshot 2024-09-29 at 10 58 21 AM" src="https://github.com/user-attachments/assets/bf73cabc-0267-408a-b551-ddcc187e52c3">

Reply:
Would it be possible to delete this pull request ? I only wanted to be helpful and I think I did things wrong